examples/flutter_gallery/test/drawer_test.dart

```diff
@@ -0,0 +1,100 @@
+// Copyright 2017 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/material.dart';
+import 'package:flutter/scheduler.dart';
+import 'package:flutter_test/flutter_test.dart';
+import 'package:flutter_gallery/gallery/app.dart';
+
+void main() {
```

Comment:
Nice to see test tests for ALL the gallery drawer features here.

Reply:
Hey, why not? :-)